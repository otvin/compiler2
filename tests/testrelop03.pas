program testrelop03(output);
var
	b:Boolean;
begin
	b:=7 < 5;
	writeln(1,'. Should be false: ',b);

	b:= 6.2 <= (9 * 8 + 7);
	writeln(2, '. Should be true: ', b);

	b:= (-37 * 1) = -37;
	writeln(3,'. Should be true: ', b);

	b:= false <= true;
	writeln(4,'. Should be true: ', b);

	b:= 72 <> (72 - 0);
	writeln(5,'. Should be false: ', b);

	b:= 123098 >= 1230;
	writeln(6, '. Should be true: ', b);

	b:= 9812.398 > -320.772;
	writeln(7, '. Should be true: ', b);
end.
