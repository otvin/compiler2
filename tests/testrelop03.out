1. Should be false: FALSE
2. Should be true: TRUE
3. Should be true: TRUE
4. Should be true: TRUE
5. Should be false: FALSE
6. Should be true: TRUE
7. Should be true: TRUE
